tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "noImplicitAny": false,
```

Comment:
Just picked up on this. Do we not want to keep this turned on? It seems like the most valuable feature of typescript otherwise it's easy to get away with writing plain Javascript, and we'd miss out on type checks that we would want